### ISSUE TITLE
Ignore __pycache__ in chalicelib on package

### ISSUE THREAD

**Issue by Sytten** [aws/chalice#1065]
Currently the `__pycache__` folder inside the `chalicelib` folder is included in the `.zip` when packaging the dependencies. While it is not critical, I don't think it should be included.

**Comment by jamesls** [comment]
Yeah, I'm leaning towards not including `__pycache__` in the deployment package.  There's been a few requests for this, and it appears that including the .pyc files don't really improve startup by much (https://github.com/aws/chalice/issues/868#issuecomment-400023344).

There's an existing PR for this: https://github.com/aws/chalice/pull/408

**Comment by Sytten** [comment]
Cool! It seems easy to do, you can assign me if you want I have time to rebase the PR and fix the concerns raised.

**Comment by Sytten** [comment]
Just pushed the PR #1068 @jamesls can you take a look? Thanks!
